daemon/logger/splunk/splunk.go

```diff
@@ -196,6 +308,12 @@ func (l *splunkLogger) Name() string {
 	return driverName
 }
 
+func (l *splunkLogger) createSplunkMessage(msg *logger.Message) splunkMessage {
+	message := *l.nullMessage
+	message.Time = fmt.Sprintf("%f", float64(msg.Timestamp.UnixNano())/1000000000)
+	return message
```

Comment:
I am not a Go expert. Are we talking about keeping it on stack and copying it or allocating it on heap and asking GC to deal with it? In case of huge demand of messages - will the first option (current implementation) work better?